quiz.js

```diff
@@ -34,11 +34,18 @@ var dingHandle = setTimeout (writeDing(), 3000);
 var words = ['short', 'medium', 'long', 'interminable', 'lengthy'];
 
 var byLength = function (A, B) {
-  // your code here
+  var i;
+    for (i=0; i < count.length; i++) {
```

Comment:
I think the .sort method tripped you up here. Google is your best friend in situations like this